app/services/ui_notifications/hosts.rb

```diff
@@ -14,12 +14,7 @@ def deliver!
       protected
 
       def audience
-        case subject.owner_type
-        when 'User'
-          ::Notification::AUDIENCE_USER
-        when 'Usergroup'
-          ::Notification::AUDIENCE_GROUP
-        end
+        ::Notification::AUDIENCE_SUBJECT if subject.owner
```

Comment:
Nice!

Reply:
Just curious, why `if subject.owner`? Isn't the audience the same and it would just resolve to nothing?

Reply:
This is a little verbose. It's not really necessary but I think, it's the cleaner solution.